_topic_maps/_topic_map.yml

```diff
@@ -3,6 +3,8 @@ Name: About OpenShift Builds
 Dir: builds
 Distros: openshift-builds
 Topics:
+- Name: Release Notes
```

Comment:
Should this title be more specific. Some other component group could also create a topic called "Release Notes" and processors might spew out errors.

Reply:
I avoid giving any title right now because what would be the name of this product is still under dicussion. So, i want to keep the content generic. I will keep this suggestion for future.